atom-jobs/docker-common.groovy

```diff
@@ -161,12 +161,30 @@ images = params.RELEASE_DOCKER_IMAGES.split(",")
 def release_images() {
     for (item in images) {
        if (item.startsWith("pingcap/")) {
-           docker.withRegistry("", "dockerhub") {
+        def harbor_tmp_image_name = "hub.pingcap.net/image-sync/" + item
```

Comment:
harbor_tmp_image_name  like `hub.pingcap.net/image-sync/pingcap/xxx`, should remove  `pingcap`